src/ImGui.NET/ImGui.cs

```diff
@@ -896,7 +940,7 @@ public static void SameLine()
             ImGuiNative.igSameLine(0, 0);
         }
 
-        public static void SameLine(float localPositionX, float spacingW)
+        public static void SameLine(float localPositionX = 0, float spacingW = -1.0f)
```

Comment:
Yes, I'd agree with you there. Go ahead and remove it and just leave the one with default parameters.